### PR TITLE
rsc: Refactored/exported some fields and methods

### PR DESCRIPTION
**Pull request by Jonatino**
Renamed some fields/methods regarding cache logic in the runescape-client project.

### CODE REVIEW

**Comment by Jonatino** [comment]
Need to remove static field imports inside RS interfaces.

File: runescape-client/src/main/java/Login.java [hunk 1]
```diff
@@ -230,4 +230,4 @@ static int method2223(int var0, Script var1, boolean var2) {
 			return 1; // L: 1317
 		}
 	}
-}
+}
```
Comment (Lucwousin) [inline runescape-client/src/main/java/Login.java:233]: Revert this file

File: runescape-client/src/main/java/WorldMapRectangle.java [hunk 2]
```diff
@@ -184,4 +184,4 @@ static final String formatItemStacks(int var0) {
 			return var1.length() > 6 ? " " + FileSystem.colorStartTag(16777215) + var1.substring(0, var1.length() - 4) + "K" + " " + " (" + var1 + ")" + "</col>" : " " + FileSystem.colorStartTag(16776960) + var1 + "</col>"; // L: 9909 9910
 		}
 	}
-}
+}
```
Comment (Lucwousin) [inline runescape-client/src/main/java/WorldMapRectangle.java:187]: Revert this file

File: runescape-client/src/main/java/class170.java [hunk 3]
```diff
@@ -2,4 +2,4 @@
 
 @ObfuscatedName("ff")
 public interface class170 {
-}
+}
```
Comment (Lucwousin) [inline runescape-client/src/main/java/class170.java:5]: Revert this file

File: runescape-client/src/main/java/class13.java [hunk 4]
```diff
@@ -340,4 +340,4 @@ static void method122() {
 		}
 
 	} // L: 11640
-}
+}
```
Comment (Lucwousin) [inline runescape-client/src/main/java/class13.java:343]: Revert this file

File: runescape-client/src/main/java/class171.java [hunk 5]
```diff
@@ -4,20 +4,22 @@
 import net.runelite.mapping.ObfuscatedSignature;
 
 @ObfuscatedName("fd")
-public class class171 implements class170 {
+public class class171 implements class170
+{
```
Comment (Lucwousin) [inline runescape-client/src/main/java/class171.java:8]: This bracket is supposed to go next to the class declaration but it doesn't really matter thát much

File: runescape-client/src/main/java/class224.java [hunk 6]
```diff
@@ -26,7 +29,8 @@
 		descriptor = "(II)V",
 		garbageValue = "2091444199"
 	)
-	static final void method4232(int var0) {
+	@Export("redrawMiniMapIcons")
```
Comment (Lucwousin) [inline runescape-client/src/main/java/class224.java:32]: I think this does more than just draw icons - if I'm correct, it draws the entire minimap :P

Maybe `createMinimapSprite`?

**Review by Lucwousin — CHANGES_REQUESTED** [review]

**Pull request merged**